docs/internals/releases.md

```diff
@@ -40,13 +40,34 @@ synchronization:
   is pushed to <https://github.com/tigerbeetle/tigerbeetle-go>
 - if publishing to all registries were successfully, the release is marked as non-draft.
 
-All publishing keys are stored as GitHub Actions secrets.
+All publishing keys are stored as GitHub Actions in the `release` environment.
 
 ## Triggering a Release
 
-Releases are triggered manually, on Monday, by pushing the the current tip of `origin/main` to
+Releases are triggered manually, on Monday, by pushing a commit from `origin/main` to
 `origin/release`. This triggers the `release.yml` workflow. The workflow doesn't run unit-tests,
-[not rocket science](https://graydon2.dreamwidth.org/1597.html) rule is assumed.
+[not rocket science](https://graydon2.dreamwidth.org/1597.html) rule is assumed. The workflow
+requires an approval of at least one other person.
+
+## Changelog
+
+Before triggering a release, update CHANGELOG.md with the changes from the last release. Merge
+commit for the changelog PR is what is pushed to the `release` branch.
```

Comment:
```suggestion
Before triggering a release, update CHANGELOG.md with the changes since the last release. The merge
commit for the changelog PR is what is pushed to the `release` branch.
```